src/main/java/org/broadinstitute/hellbender/tools/copynumber/CallCopyRatioSegments.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.caller.CalledCopyRatioSegmentCollection;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.caller.ReCapSegCaller;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.copyratio.CopyRatioCollection;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.segmentation.CopyRatioSegmentCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.CopyNumberStandardArgument;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.File;
+
+/**
+ * Calls segments as amplified, deleted or copy number neutral given files containing denoised copy ratios
+ * and a list of segments.
+ *
+ * @author David Benjamin
+ *
+ * <h3>Examples</h3>
+ *
+ * <pre>
+ * gatk-launch --javaOptions "-Xmx4g" CallCopyRatioSegments \
+ *   --denoisedCopyRatios tumor.denoisedCR.tsv \
+ *   --segments tumor.craf.seg \
+ *   --output tumor.called
+ * </pre>
+ */
+@CommandLineProgramProperties(
+        summary = "Call copy-ratio segments as amplified, deleted, or copy number neutral.",
+        oneLineSummary = "Call copy-ratio segments as amplified, deleted, or copy number neutral.",
+        programGroup = CopyNumberProgramGroup.class
+)
+@DocumentedFeature
+public final class CallCopyRatioSegments extends CommandLineProgram {
+
+    @Argument(
+            doc = "Input file containing denoised copy-ratio profile (output of DenoiseReadCounts).",
+            fullName = CopyNumberStandardArgument.DENOISED_COPY_RATIOS_FILE_LONG_NAME,
+            shortName = CopyNumberStandardArgument.DENOISED_COPY_RATIOS_FILE_SHORT_NAME
+    )
+    private File inputDenoisedCopyRatiosFile;
+
+    @Argument(
+            doc = "Input file containing copy-ratio segments (.craf.seg output of ModelSegments).",
+            fullName = CopyNumberStandardArgument.SEGMENTS_FILE_LONG_NAME,
+            shortName = CopyNumberStandardArgument.SEGMENTS_FILE_SHORT_NAME
+    )
+    protected File segmentsFile;
```

Comment:
Why is one input `private` and one `protected`?

Reply:
Good catch, should be private.  Probably creeped over from code where we made fields protected.